include/os/linux/zfs/sys/trace_vdev.h

```diff
@@ -24,8 +24,8 @@
 
 /*
  * If tracepoints are available define dtrace_probe events for vdev
- * related probes.  Definitions in usr/include/trace.h will map
- * DTRACE_PROBE* calls to tracepoints.
+ * related probes.  Definitions in usr/include/os/linux/spl/trace_impl.h
```

Comment:
Do we want to drop the usr/ prefix here?

Reply:
There doesn't seem to be a file actually named trace_impl.h

Reply:
yeah, I had to rename it again because of tracing on linux involves some self referential voodoo that requires the file with the defines be named trace.h I'll fix the comment once I get more reviews. Thanks.